library/Ivoz/Kam/Domain/Service/TrunksCdr/TrunksCdrLifecycleServiceCollection.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace Ivoz\Kam\Domain\Service\TrunksCdr;
+
+use Ivoz\Core\Domain\Service\LifecycleServiceCollectionInterface;
+use Ivoz\Core\Domain\Service\LifecycleServiceCollectionTrait;
+
+/**
+ * @codeCoverageIgnore
+ */
+class TrunksCdrLifecycleServiceCollection implements LifecycleServiceCollectionInterface
+{
+    use LifecycleServiceCollectionTrait;
+
+    public static $bindedBaseServices = [
+        "on_domain_event" =>
```

Comment:
also constant here

Reply:
We don't use constants on bindedBaseServices keys because it's easier to read this way and because it's auto generated (no typo possibility).